examples/sagaexecutor/components/secrets.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: local-secret-store 
+spec:
+  type: secretstores.local.file
+  version: v1
+  metadata:
+  - name: secretsFile
+    value: /Users/stevef/dev/sagaexecutor/components/secrets.json
```

Comment:
This absolute path to the secrets file appears to be problematic for those attempting to use this. A change required to this file is not documented so I believe it should be relative.

Reply:
> This absolute path to the secrets file appears to be problematic for those attempting to use this. A change required to this file is not documented so I believe it should be relative.

I will look at this later when I have recreated a K8s system to test it against.

Reply:
> > This absolute path to the secrets file appears to be problematic for those attempting to use this. A change required to this file is not documented so I believe it should be relative.
> 
> I will look at this later when I have recreated a K8s system to test it against.

This absolute path was required for local testing and relative paths wouldn't let me load the component. I have added some code to the setup local.sh file to update this hard path to the one required so that should now work for you.